src/message/messageActionSheet.js

```diff
@@ -43,6 +43,11 @@ type ButtonDescription = {
 
 const isAnOutboxMessage = (message: Message | Outbox): boolean => message.isOutbox;
 
+const getRawMessageContent = async (auth: Auth, message: Message | Outbox) =>
+  isAnOutboxMessage(message) /* $FlowFixMe: then really type Outbox */
```

Comment:
This should be fixable simply by testing `message.isOutbox` inline, rather than calling `isAnOutboxMessage`.